### PR TITLE
#3956 Hover effect is absent on billing options

### PR DESCRIPTION
**Pull request by iman-sujamiko**
**Related issue(s):**
* Fixes https://github.com/scandipwa/scandipwa/issues/3956

**Problem:**
* Hover effect is absent on billing options

**In this PR:**
* changed .Field_type_checkbox pointer-events from none to auto
* wraped billing options title in label pointed to billing options input checkbox

### CODE REVIEW

**Review by Sliceem — APPROVED** [review]

**Review by AleksandrsKondratjevs — APPROVED** [review]

File: packages/scandipwa/src/component/CheckoutPayment/CheckoutPayment.style.scss [hunk 1]
```diff
@@ -48,6 +48,6 @@
 
     .Field_type_checkbox {
         margin-block-start: 0;
-        pointer-events: none;
+        pointer-events: auto;
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/CheckoutPayment/CheckoutPayment.style.scss:51]: If auto then do we even need it ? I guess it is auto by default.

**Comment by aleksandrakorolova** [comment]
Please, add hover effect for text (name of option) too + change cursor to 'hand' when hovering the mouse over text option

**Comment by aleksandrakorolova** [comment]
Still, the hover effect is missing for text. It doesn't become orange. Please, change text color when hovering on it as it works in Layered navigation on PLP
Screenshot 2022-01-10 at 08 29 09

File: packages/scandipwa/src/component/Field/Field.style.scss [hunk 2]
```diff
@@ -54,6 +54,9 @@
 
         &:hover {
             cursor: pointer;
+            @include desktop {
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/Field/Field.style.scss:57]: Add space between

**Review by AleksandrsKondratjevs — CHANGES_REQUESTED** [review]

**Review by AleksandrsKondratjevs — APPROVED** [review]

**Pull request merged**